test/controllers/static_pages_controller_test.rb

```diff
@@ -18,4 +18,4 @@ class StaticPagesControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
```

Comment:
- Cho toàn bộ folder test vài .gitignore
- xóa cached folder test bằng lệnh git rm --cached test/*